File: 461_Hamming_Distance.java
public class Solution {
    public int hammingDistance(int x, int y) {
        int z=x^y;
        return Integer.bitCount(z);
    }
}


public class Solution {
    public int hammingDistance(int x, int y) {
        int z=x^y;
        int count=0;
        while(z!=0){
            if((z>>1)<<1!=z) count++;
            z=z>>1;
        }
        return count;
    }
}

File: 461_Hamming_Distance.py
class Solution(object):
    def hammingDistance(self, x, y):
        """
        :type x: int
        :type y: int
        :rtype: int
        """
        a= x^y
        return bin(a).count('1')

class Solution(object):
    def hammingDistance(self, x, y):
        """
        :type x: int
        :type y: int
        :rtype: int
        """
        a= x^y
        result=0
        while(a):
            if (a>>1)<<1 != a:
                result+=1
            a=a>>1
        return result